cmd/forwarder/http.go

```diff
@@ -147,6 +150,18 @@ func (s *httpServer) Run() error {
 			defer body.Close()
 		}
 
+		buf := new(bytes.Buffer)
```

Comment:
Prefer `var buf bytes.Buffer`. This is somewhat of a personal thing, but aligns with what is shown in the official docs/examples. Also IME, `new()` is seldom used and a little jarring.